### ISSUE TITLE
format_spec_file mangles some spec files

### ISSUE THREAD

**Issue by plusky** [openSUSE/spec-cleaner#108]
When building libmicrohttpd (see [sr#390041](https://build.opensuse.org/request/show/390041), running `osc service run format_spec_file` causes following:
```
Index: libmicrohttpd.spec
===================================================================
--- libmicrohttpd.spec  (revision 14e1e9d0482e4a3d5a3e45feca754cda)
+++ libmicrohttpd.spec  (working copy)
@@ -31,17 +31,36 @@
 # PATCH-WORKAROUND-OPENSUSE: the threads have a problem deadlocking (in OBS)
 Patch1:         disable-stalling-test.patch
 Patch2:         libmicrohttpd_test_data.patch
-BuildRequires:  curl
-BuildRequires:  file-devel
-BuildRequires:  libgcrypt-devel >= 1.2.4
-BuildRequires:  libtool
-BuildRequires:  makeinfo
-BuildRequires:  openssl-devel
-BuildRequires:  pkg-config
-BuildRequires:  socat
-BuildRequires:  pkgconfig(gnutls)
-BuildRequires:  pkgconfig(libcurl)
-BuildRequires:  pkgconfig(libtasn1)
+BuildRequires:
+BuildRequires:  (
+BuildRequires:  )
+BuildRequires:  -
+BuildRequires:  .
+BuildRequires:  1
+BuildRequires:  2
+BuildRequires:  4
+BuildRequires:  =
+BuildRequires:  >
+BuildRequires:  a
+BuildRequires:  b
+BuildRequires:  c
+BuildRequires:  d
+BuildRequires:  e
+BuildRequires:  f
+BuildRequires:  g
+BuildRequires:  i
+BuildRequires:  k
+BuildRequires:  l
+BuildRequires:  m
+BuildRequires:  n
+BuildRequires:  o
+BuildRequires:  p
+BuildRequires:  r
+BuildRequires:  s
+BuildRequires:  t
+BuildRequires:  u
+BuildRequires:  v
+BuildRequires:  y
```
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build

**Comment by plusky** [comment]
spec-cleaner and service are both 0.8 from openSUSE_Tools project

**Comment by scarabeusiv** [comment]
ow crap.

**Comment by scarabeusiv** [comment]
Revert this https://github.com/openSUSE/spec-cleaner/commit/ad95a8e70c930d29aaee63220356761f7aa18d4b

**Comment by scarabeusiv** [comment]
it seems it was there on purpose and we obviously should add test for it. Wanna write it?

**Comment by scarabeusiv** [comment]
Code updated and version 0.8.1 released. Now pending testcase so this won't happen in future.

**Comment by scarabeusiv** [comment]
Fixed by d74a93864a9b7ec4faf2778c76fd0c413c46aaee
